e2e_tests/tests/command/test_tensorboard.py

```diff
@@ -223,6 +224,35 @@ def test_tensorboard_inherit_image_pull_secrets() -> None:
     assert ips == exp_secrets, (ips, exp_secrets)
 
 
+@pytest.mark.e2e_cpu
+def test_delete_tensorboard_for_experiment() -> None:
+    """
+    Start a random experiment, start a TensorBoard instance pointed to
+    the experiment, delete tensorboard and verify deletion.
+    """
+    config_obj = conf.load_config(conf.fixtures_path("no_op/single-one-short-step.yaml"))
+    experiment_id = exp.run_basic_test_with_temp_config(config_obj, conf.fixtures_path("no_op"), 1)
+
+    body = bindings.v1LaunchTensorboardRequest(
+        experimentIds=[experiment_id],
+    )
+
+    bindings.post_LaunchTensorboard(session=determined_test_session(), body=body)
+
+    command = ["det", "tensorboard", "delete", str(experiment_id)]
+    subprocess.run(command, universal_newlines=True, stdout=subprocess.PIPE, check=True)
+
+    # Check if Tensorboard files are deleted
+    command = ["find", "-P", "/tmp/determined-cp/", "-name", "tensorboard*", "-type", "d"]
+    res = subprocess.run(command, universal_newlines=True, stdout=subprocess.PIPE, check=True)
+
+    path = res.stdout[res.stdout.find("//") + 2 :].strip("\n")
+    command = ["ls", "/tmp/determined-cp/" + path + "/experiment"]
+    subprocess.run(command, universal_newlines=True, stdout=subprocess.PIPE, check=True)
```

Comment:
personal preference, can we just write python code that does this instead?

Reply:
using pathlib library now